packages/asset-service/src/generateAssetData/generateAssetData.ts

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs'
+import { baseAssets } from './baseAssets'
+import { getTokens } from './ethTokens'
+import { BaseAsset, NetworkTypes } from '../types'
+import { AssetService } from '../'
+
+const generateAssetData = async () => {
+  const generatedAssetData = await Promise.all(
```

Comment:
Are we okay with this failing if any of the promises fail?

We may want to do a `Promise.allSettled` and then filter the results to the successful results.